### PR TITLE
move tunneler out of generic

### PR DESCRIPTION
**Pull request by deads2k**
moves the tunneler out of genericapiserver and under master where its used.

@sttts

### CODE REVIEW

File: pkg/master/tunneler/tunneler.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package genericapiserver
+package tunneler
```
Comment (sttts) [inline pkg/master/tunneler/tunneler.go]: nit: -> ssh.go

File: cmd/kube-apiserver/app/server.go [hunk 2]
```diff
@@ -141,20 +142,20 @@ func Run(s *options.ServerRunOptions) error {
...
+		// Set up the nodeTunneler
 		// TODO(cjcullen): If we want this to handle per-kubelet ports or other
 		// kubelet listen-addresses, we need to plumb through options.
 		healthCheckPath := &url.URL{
 			Scheme: "https",
 			Host:   net.JoinHostPort("127.0.0.1", strconv.FormatUint(uint64(s.KubeletConfig.Port), 10)),
 			Path:   "healthz",
 		}
-		tunneler = genericapiserver.NewSSHTunneler(s.SSHUser, s.SSHKeyfile, healthCheckPath, installSSH)
+		nodeTunneler = tunneler.NewSSHTunneler(s.SSHUser, s.SSHKeyfile, healthCheckPath, installSSH)
```
Comment (sttts) [inline cmd/kube-apiserver/app/server.go]: nit: tunneler.New

File: cmd/kube-apiserver/app/server.go [hunk 3]
```diff
@@ -123,12 +124,12 @@ func Run(s *options.ServerRunOptions) error {
...
 
-	// Setup tunneler if needed
-	var tunneler genericapiserver.Tunneler
+	// Setup nodeTunneler if needed
+	var nodeTunneler tunneler.Tunneler
 	var proxyDialerFn utilnet.DialFunc
 	if len(s.SSHUser) > 0 {
 		// Get ssh key distribution func, if supported
-		var installSSH genericapiserver.InstallSSHKey
+		var installSSH tunneler.InstallSSHKey
```
Comment (sttts) [inline cmd/kube-apiserver/app/server.go]: nit: installSSHKey

**Comment by sttts** [comment]
some nit. otherwise, lgtm

**Comment by deads2k** [comment]
@k8s-bot gci gce e2e test this

**Pull request merged**